### PR TITLE
Model list should be curated in production

### PR DESCRIPTION
**Pull request by mrerrormessage**
this was useful during development, but will probably just confuse/confound users

### CODE REVIEW

**Comment by TheBizzle** [comment]
Don't these changes render the status CSS classes and the icons pointless?  I don't think it's a bad thing if it does, but then those things should be cleaned up.

**Comment by mrerrormessage** [comment]
I suppose they make "not_compiling" pretty pointless, but do we want to keep the "unknown" around, in case some models aren't compiled? I wasn't sure about whether we wanted to keep showing the "not_compiling" models in development or not, I can see pros and cons on both sides - what are your thoughts?

**Comment by TheBizzle** [comment]
I hadn't thought about `unknown`.  You're right to point out that that should stick around (which means that `compiling` should, too).

Here's my ideal case:

All of the icons continue to live on only in Development Mode.  On the static pages that we distribute, there should be no icons, and only models that compile should be shown in the list.

**Comment by mrerrormessage** [comment]
That makes sense. I was a bit hesitant to do that at first because it requires pretty different behavior in development mode, but I agree that seems like the best way to go. I'll modify.

File: app/assets/javascripts/models.coffee [hunk 1]
```diff
@@ -2,17 +2,27 @@ exports.bindModelChooser = (container, selectionChanged) ->
 
   PUBLIC_PATH_SEGMENT_LENGTH = "public/".length
 
-  adjustModelName = (modelName) ->
+  adjustModelPath = (modelName) ->
     modelName.substring(PUBLIC_PATH_SEGMENT_LENGTH, modelName.length)
 
+  modelDisplayName = (modelName) ->
+    adjustedModelPath = adjustModelPath(modelName)
+    if adjustedModelPath.substring(0, 10) == "modelslib/"
```
Comment (Idloj) [inline app/assets/javascripts/models.coffee]: Isn't there a `startsWith` function?

Comment (TheBizzle) [inline app/assets/javascripts/models.coffee]: Unfortunately, support for that is first being added in ES6.  The sake of compatibility, our project targets ES5.

**Comment by mrerrormessage** [comment]
This should be ready for re-review.

**Pull request merged**